File: skills/astra-toolkit/clients/python/examples/tables-list-names-keyspace.py
import os
from astrapy import DataAPIClient

# Get a database
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)

# List table names
result = database.list_table_names(keyspace="**KEYSPACE_NAME**")
print(result)

File: skills/astra-toolkit/clients/java/examples/tables-list-names-keyspace.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.databases.Database;
import com.datastax.astra.client.tables.commands.options.ListTablesOptions;
import java.util.List;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient(System.getenv("ASTRA_DB_APPLICATION_TOKEN")).getDatabase(System.getenv("ASTRA_DB_API_ENDPOINT"));

    // List table names
    ListTablesOptions options = new ListTablesOptions().keyspace("**KEYSPACE_NAME**");
    List<String> result = database.listTableNames(options);

    System.out.println(result);
  }
}
